deps.edn

```diff
@@ -108,34 +108,34 @@
   ;; clojure -M:repl/cider
   :repl/cider
   {:extra-deps {nrepl/nrepl       {:mvn/version "0.9.0"}
-                cider/cider-nrepl {:mvn/version "0.28.3"}}
+                cider/cider-nrepl {:mvn/version "0.28.4"}}
    :main-opts  ["-m" "nrepl.cmdline"
                 "--middleware" "[cider.nrepl/cider-middleware]"
                 "-i"]}
 
   :repl/cider-refactor
   {:extra-deps {nrepl/nrepl                   {:mvn/version "0.9.0"}
-                cider/cider-nrepl             {:mvn/version "0.28.3"}
+                cider/cider-nrepl             {:mvn/version "0.28.4"}
                 refactor-nrepl/refactor-nrepl {:mvn/version "3.5.2"}}
    :main-opts  ["-m" "nrepl.cmdline"
                 "--middleware" "[refactor-nrepl.middleware/wrap-refactor,cider.nrepl/cider-middleware]"
                 "-i"]}
 
   ;; clojure -M:repl/cider-cljs
   :repl/cider-cljs
-  {:extra-deps {org.clojure/clojurescript       {:mvn/version "1.10.773"}
+  {:extra-deps {org.clojure/clojurescript       {:mvn/version "1.11.54"}
```

Comment:
TODO: test version of ClojureScript works okay.  Some recent versions of ClojureScript were causing issues and kept back to 1.10.773